File: NEA algorithm/algorithm.js
function generateSquare() {
    let square = [];
    for (let i = 0; i < 6; i++) {
        square[i] = [];
        for (let j = 0; j < 6; j++) {
            square[i][j] = ' ';
        }
    }
    for (let i = 0; i < 20; i++) {
        let row = Math.floor(Math.random() * 6);
        let col = Math.floor(Math.random() * 6);
        square[row][col] = 'W';
    }
    return square;
}

function printSquare(square) {
    for (let i = 0; i < square.length; i++) {
        let row = square[i];
        let rowStr = '';
        for (let j = 0; j < row.length; j++) {
            rowStr += row[j];
        }
        document.write(rowStr + "<br>");
    }
}

let square = generateSquare();
printSquare(square);
document.write("Finished executing code");

console.log("done")

File: NEA algorithm/algorithm.py
import random

def generate_square():
    square = [[' ' for i in range(6)] for j in range(6)]
    for i in range(20):
        row = random.randint(0, 5)
        col = random.randint(0, 5)
        square[row][col] = 'W'
    return square

def print_square(square):
    print('-' * 13)
    for row in square:
        print('|', end='')
        for col in row:
            print(col, end=' |')
        print()
        print('-' * 13)

square = generate_square()
print_square(square)
print('hi')